snakefile.py

```diff
@@ -172,13 +199,12 @@ def tool(name):
     log: os.path.join(LOG_DIR, 'bwa_index.log')
     shell: """mkdir -p {INDEX_DIR};
 ln -sf {input} {INDEX_DIR};
-cd {INDEX_DIR};
 {BWA_EXEC} index {output.ref} >> {log} 2>&1"""
 
 
 rule bwa_align:
     input:
-        fastq = [os.path.join(TRIMMED_READS_DIR, "{sample}_trimmed_R1.fastq"), os.path.join(TRIMMED_READS_DIR, "{sample}_trimmed_R2.fastq")],
+        fastq = [os.path.join(TRIMMED_READS_DIR, "{sample}_trimmed_R1.fastq.gz"), os.path.join(TRIMMED_READS_DIR, "{sample}_trimmed_R2.fastq.gz")],
```

Comment:
looks like we could have just passed the raw files here.  Samtools doesn't seem to mind.